saged/models.py

```diff
@@ -227,11 +228,18 @@ def __init__(self,
         l2_penalty: The inverse of the degree to which weights should be penalized
         solver: The method to use to optimize the loss
         """
-        self.model = sklearn.linear_model.LogisticRegression(random_state=seed,
-                                                             class_weight='balanced',
-                                                             penalty='l2',
-                                                             C=l2_penalty,
-                                                             solver=solver)
+        if penalty_type == 'none':
+            self.model = sklearn.linear_model.LogisticRegression(random_state=seed,
+                                                                 class_weight='balanced',
+                                                                 penalty='none',
```

Comment:
Do you want to add an option for `solver` here? Looks like the option will only take effect if you pass a `penalty_type` other than `'none'`.